postman/config.yml

```diff
@@ -0,0 +1,39 @@
+# Name of the quickstart (required)
+name: postman
+
+# Displayed in the UI (required)
+title: Postman
+
+# Long-form description of the quickstart (required)
+description: |
+  Observe the performance of your APIs using Postman and New Relic
+
+# Displayed in search results and recommendations. Summarizes a quickstarts functionality.
+summary: |
+  Observe the performance of your APIs using Postman and New Relic
```

Comment:
Can we make this summary more product focused and a have more valuable user focused content?